plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb

```diff
@@ -441,6 +441,13 @@ def worker_is_unstable
       out.must_include "UNSTABLE"
     end
 
+    it "ignores allowed percentage of failures" do
```

Comment:
need a test with your example listed in the description

Reply:
changed it to 1/3 since that is the lowest I can do without messing with the whole test setup